dipy/tracking/fbcmeasures.pyx

```diff
@@ -261,45 +259,47 @@ cdef class FBCMeasures:
             else:
                 print("No OpenMP...")
 
+        cdef cnp.npy_intp line_id_ng = 0
+        cdef int point_id_ng = 0, line_id2_ng = 0, point_id2_ng = 0
```

Comment:
I do not understand why you need to rename the variables and change the location of the declaration. Why you did not keep and initialize the previous one?

Reply:
These keep throwing warnings, so the change does not certainly help. I was hypothesizing that the code used when the GIL is released required variables that were different from the code when the GIL is hold. But the assumption seems to be inaccurate as for the results. So this is likely to be reversed or changed depending on your investigations.

Reply:
Reverted in 10032f0.